tests: updated the documentation to run spread tests using external backend (#11341)
snapcore/snapd: tests/external-backend.md

* From the snapd project's root execute the script to setup ssh access to the
instance:
```
$ ./tests/lib/external/prepare-ssh.sh <instance_ip> <instance_port> <launchpad_id>
```
The default values for ip and port are `localhost`, `8022`. This script assumes that
the user created by console-conf has the same name as the user executing the
script, if that's not the case you can pass the created username as a third argument
to the script.

* From the snapd project's root execute the suite selecting the type of system of

> the instance, in the spread.yaml file are listed all the supported systems:

Reviewer: ```suggestion
the instance (spread.yaml file lists all supported systems) by executing the command:
```
